src/main/java/com/icthh/xm/tmf/ms/resource/client/OAuth2InterceptedFeignConfiguration.java

```diff
@@ -2,11 +2,12 @@
 
 import feign.RequestInterceptor;
 import io.github.jhipster.security.uaa.LoadBalancedResourceDetails;
-import java.io.IOException;
 import org.springframework.cloud.security.oauth2.client.feign.OAuth2FeignRequestInterceptor;
 import org.springframework.context.annotation.Bean;
 import org.springframework.security.oauth2.client.DefaultOAuth2ClientContext;
 
+import java.io.IOException;
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version